packages/mobile/src/services/offline-downloader/offline-download-queue.ts

```diff
@@ -79,13 +138,52 @@ export const startDownloadWorker = async () => {
   queue.start()
 }
 
+export const cancelQueuedCollectionDownloads = async (
```

Comment:
is this called anywhere?

Reply:
ooh yep we missed that. Added the call in removeCollectionDownload